backend/src/main/java/de/neuefische/backend/modelle/LocationDTO.java

```diff
@@ -1,19 +1,132 @@
 package de.neuefische.backend.modelle;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import org.springframework.format.annotation.DateTimeFormat;
 
+import java.math.BigDecimal;
+import java.time.LocalDate;
 
+public class LocationDTO {
```

Comment:
Hier wäre in Zukunft der `@AllArgsConstructor` von Lombok in Zukunft vlt für dich praktisch

Reply:
Oh, good job!